EventListener/RequestExecutedListener.php

```diff
@@ -29,11 +34,13 @@ class RequestExecutedListener
     /**
      * RequestExecutedListener constructor.
      *
+     * @param array         $parameters
      * @param ApiLogUtility $apiLogUtility
      * @param EmailUtility  $emailUtility
      */
-    public function __construct(ApiLogUtility $apiLogUtility, EmailUtility $emailUtility)
+    public function __construct(array $parameters, ApiLogUtility $apiLogUtility, EmailUtility $emailUtility)
```

Comment:
It's Breaking change! It should be in branch `2.0.0-alpha1` and not modified in `1.3.X` in case of anybody extends this listener